components/runtime/squeak4/Makefile

```diff
@@ -0,0 +1,67 @@
+#
+# This file and its contents are supplied under the terms of the
+# Common Development and Distribution License ("CDDL"). You may
+# only use this file in accordance with the terms of the CDDL.
+#
+# A full copy of the text of the CDDL should have accompanied this
+# source. A copy of the CDDL is also available via the Internet at
+# http://www.illumos.org/license/CDDL.
+#
+
+#
+# Package Makefile for the "Classic VM" Squeak Smalltalk V4 system
+# See http://squeak.org and http://squeakvm.org
+#
+# The squeak4 vm can only run Squeak4 images; it will not work with 
+# Squeak5 images; squeak5 uses  different technology called "Cog Spur"
+#
+# squeak4 uses cmake and a custom configure script (not GNU autoconf)
+# in order to build this package, "cmake" must be installed
+#
+
+include ../../../make-rules/shared-macros.mk
+
+COMPONENT_NAME=		squeak4
+COMPONENT_VERSION=	4.16.7
+COMPONENT_SUMMARY=	The Squeak V4 Smalltalk Virtual Machine
+COMPONENT_PROJECT_URL=	http://www.squeak.org
+COMPONENT_ARCHIVE_URL=  http://squeakvm.org
+COMPONENT_FMRI=		runtime/squeak4
+COMPONENT_CLASSIFICATION=	Development/Smalltalk
+SVN_REPO=		http://squeakvm.org/svn/squeak/trunk/
+SVN_REV=		3775
+SVN_HASH=  sha256:1f40ae6a9ad05739c07d63f4e67bfd6afacbe76c4deb2802bba1be7332f373a7
+COMPONENT_LICENSE=      LICENSE
+
```

Comment:
You still need to set correct COMPONENT_LICENSE and COMPONENT_LICENSE_FILE.
For COMPONENT_LICENSE I am not sure what to set it to as the license file doesn't give a name to the licenses. Probably something like "various Open Source, see license file and individual soure files".

When everything is done please rebase your repository to latest OpenIndiana/oi-hipster and then squash all your commit belonging to this PR into one. Maybe alp has some more comments.

Reply:
Based on http://wiki.squeak.org/squeak/159 and http://wiki.squeak.org/squeak/933  I have to use "MIT" as LICENSE.   Regarding license files, I can also copy those URL's as COMPONENT_LICENSE_URL if that exists ??

Reply:
No, our build system doesn't know about COMPONENT_LICENSE_URL but only COMPONENT_LICENSE and COMPONENT_LICENSE_FILE. You can manually download and add the license information to your PR, though. If you do it set COMPONENT_LICENSE_FILE accordingly.